components/match2/wikis/teamfortress/match_group_input_custom.lua

```diff
@@ -270,43 +166,13 @@ end
 -- Parse extradata information
 ---@param map table
 ---@return table
-function mapFunctions.getExtraData(map)
-	map.extradata = {
+function MapFunctions.getExtraData(map)
+	return {
 		comment = map.comment,
 		header = map.header,
 		logstf = Logic.isNotEmpty(map.logstf) and ('https://logs.tf/' .. map.logstf) or nil,
 		logstfgold = Logic.isNotEmpty(map.logstfgold) and ('https://logs.tf/' .. map.logstfgold) or nil,
```

Comment:
Are these Links? Put them in match's links instead, you can grab inspiration from dota2 for example

Reply:
yeah these are links but it's being used inside the map instead 
![image](https://github.com/user-attachments/assets/1928d70e-9978-48ed-9fd4-95f808af2d2f)

otherwise it would be like 
```
|logstf1= |logstfgold1=
|logstf2= |logstfgold2=
|logstf3= |logstfgold3=
|logstf4= |logstfgold4=
|logstf5= |logstfgold5=
``` outside the map

Reply:
That's fine, you can still store the in the match links :)

Reply:
The actual input for both logs do end up having  different id input despite it's being within the same map

on top of that in a single map: there might be a logstf, but not the gold one as well (gold is like Overtime section)

![image](https://github.com/user-attachments/assets/854db55b-7b47-4afa-8779-e1c01288718f)

Reply:
I think I might just ended up broke the module instead https://liquipedia.net/teamfortress/index.php?title=Module%3AMatchGroup%2FInput%2FCustom%2Fdev&type=revision&diff=128666&oldid=128662 

I moved the other LINK PREFIX into that function as well
https://liquipedia.net/teamfortress/User:Hesketh2

Reply:
not sure what went wrong but my goal is to keep this work 
![image](https://github.com/user-attachments/assets/a58c2129-0201-4835-84f8-ebcf9acc0c82)
 (logstf and logstf gold being within the map)

but not something like this 
![image](https://github.com/user-attachments/assets/d3ef8b7e-7fdf-4ee2-b4f8-961fb9f00b0e)